yarpl/test/FlowableTest.cpp

```diff
@@ -593,5 +593,20 @@ TEST(FlowableTest, SubscribeMultipleTimes) {
   EXPECT_EQ(results[4], std::vector<int64_t>({1, 2, 3, 4, 5}));
 }
 
+TEST(FlowableTest, ConsumerThrows_OnNext) {
```

Comment:
can we add similar test for onNext and onComplete with throwing exception?

Reply:
let me do that.